backend/src/api.ts

```diff
@@ -323,8 +323,8 @@ loginCheckedDelete('/team-event', async (_, user) => {
   return {};
 });
 loginCheckedPost('/team-event-attendance', async (req, user) => {
-  await requestTeamEventCredit(req.body.request, user);
```

Comment:
Very minor nit: but you can rewrite this a little shorter:

```
loginCheckedPost('/team-event-attendance', async (req, user) => {
  attendance: createdAttendance
}
```
Let's change the `attendance` field to be `teamEventAttendance` to be consistent with the other endpoints?